tdrs-frontend/src/components/Routes/Routes.js

```diff
@@ -23,17 +22,14 @@ const Routes = () => {
       <Route exact path="/login">
         <LoginCallback />
       </Route>
-      <PrivateRoute exact title="Welcome to TDP" path="/welcome">
-        <Welcome />
+      <PrivateRoute exact title="Welcome to TDP" path="/home">
+        <Home />
       </PrivateRoute>
       <PrivateRoute exact title="TANF Data Files" path="/data-files">
         <Reports />
       </PrivateRoute>
-      <PrivateRoute exact title="Request Access" path="/edit-profile">
-        <EditProfile />
-      </PrivateRoute>
-      <PrivateRoute exact title="Request Submitted" path="/request">
-        <Request />
```

Comment:
As this is captured in the new Home page, this route and component are deprecated.